frontend/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            URL Shortener
+        </title>
+    </head>
+    <body>
+        <h1>
+            URL Shortener
+        </h1>
+        <form>
+            <label for="url-input"> Enter your URL:</label><br>
+            <input type="text" id="url-input" name="url-input"><br>
+            <button type="button" onclick="shortenUrl()">continue</button>
+        </form>
+        <div id="result-container" style="display: none;">
+            <h3>Shortened URL:</h3>
+            <a id="short_url" href="" target="_blank"></a>
+        </div>        
+        <script>
+            function shortenUrl(){
+                const longUrl = document.getElementById("url-input").value;
+                fetch("http://127.0.0.1:5000/shorten_url",{
+                    method: "POST",
+                    body: JSON.stringify({long_url:longUrl}),
+                    headers:{
+                        "Content-Type": "application/json"
+                    }
+                })
+                .then(response => response.json())
+                .then(data => {
+                    document.getElementById("short_url").innerHTML = data.short_url;
+                    document.getElementById("short_url").href = data.short_url;
+                    document.getElementById("result-container").style.display = "block";
+                })
+                .catch(error => console.error(error));
```

Comment:
For later:
Good to catch errors, but you could show errors on web page as well,